lumigator/python/mzai/backend/backend/tests/conftest.py

```diff
@@ -6,6 +7,7 @@
 from fastapi import FastAPI
 from fastapi.testclient import TestClient
 from mypy_boto3_s3 import S3Client
+from requests_mock import Mocker
```

Comment:
Maybe `import requests_mock` directly to use Mocker in a qualified way, so we know what kind of Mock we're using exactly?